pkg/services/rpcsrv/subscription_test.go

```diff
@@ -418,6 +433,82 @@ func TestFilteredSubscriptions(t *testing.T) {
 	}
 }
 
+func TestFilteredMempoolSubscriptions(t *testing.T) {
+	priv0 := testchain.PrivateKeyByID(0)
+	goodSender := priv0.GetScriptHash()
```

Comment:
Take a look at TestFilteredNotaryRequestSubscriptions and add a comment why this test is not an extension of TestFilteredSubscripitons.